pyproject.toml

```diff
@@ -0,0 +1,118 @@
+[build-system]
+requires = [
+  "setuptools>=61",
+  "setuptools_scm[toml]>=3.4.3",
+  "wheel",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "gwsumm"
+description = "A python toolbox used by the LIGO Scientific Collaboration for detector characterisation"
+readme = "README.rst"
+requires-python = ">=3.9"
+authors = [
+  { name = "Alex Urban", email = "alex.urban@ligo.org" },
+  { name = "Duncan Macleod", email = "duncan.macleod@ligo.org" },
+  { name = "Evan Goetz", email = "evan.goetz@ligo.org" },
+]
```

Comment:
You should probably provide `maintainers` as well to indicate that only @eagoetz is a current maintainer, and contributions from Alex and I are 'historical'.

Reply:
Fixed